src/app/components/SummaryHighlightItem/SummaryHighlightItem.tsx

```diff
@@ -0,0 +1,31 @@
+import classes from './SummaryHighlightItem.module.css';
+
+// timestamp in seconds
+function createTimeString(timestamp: number): string {
+  const duration = Number(timestamp);
+  const minutes = Math.floor((duration % 3600) / 60);
+  const seconds = Math.floor((duration % 3600) % 60);
+
+  const minutesDisplay = minutes > 0 ? minutes : '0';
+  const secondsDisplay = seconds > 0 ? seconds : '00';
+  return `${minutesDisplay}:${secondsDisplay}`;
+}
+
+type summaryHightlightItemProp = {
```

Comment:
If you follow the code style like in the other components, it would be
```suggestion
type SummaryHightlightItemProps = {
```